src/Validator/ConstraintsSetFactory/Field/PercentFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AssoConnect\ValidatorBundle\Validator\ConstraintsSetFactory\Field;
+
+use AssoConnect\PHPPercent\Percent as PercentObject;
+use AssoConnect\ValidatorBundle\Validator\Constraints\Percent as PercentConstraint;
+use Symfony\Component\Validator\Constraints\Type;
+
+class PercentFactory implements FieldConstraintsSetFactoryInterface
+{
+    public function supports(string $type): bool
+    {
+        return 'percent' === $type;
```

Comment:
Maybe it can be implemented by Percent bundle which provides doctrine type like with PhpDate. What do you think? 
We can suggest these 2 packages in our composer file instead of requiring them.